plugins/@grouparoo/demo/src/util/shared.ts

```diff
@@ -16,6 +17,20 @@ export async function execSync(level: number, command) {
   await sharedExecSync(command);
 }
 
+export async function prettier(fileOrDirPath) {
+  // prettier format
+  const pCmd = path.resolve(
+    path.join(__dirname, "..", "..", "node_modules", ".bin", "prettier")
+  );
+  const pConfig = path.resolve(
+    path.join(__dirname, "..", "..", "..", "..", "..", ".prettierrc")
+  );
+  await execSync(
+    2,
+    `'${pCmd}' --config '${pConfig}' --write '${fileOrDirPath}'`
+  );
+}
+
```

Comment:
Ah! I see why you use prettier now :D
It might be better to use the built-in API rather than shelling out - https://prettier.io/docs/en/api.html

Reply:
I don't see a way to do the whole directory in the API

Reply:
Try out the `glob` package to get the list of files in a directory. 

For code customers might run, you can't rely on the path structure you are using here.  
`node_modules/.bin` doesn't exist on windows for example, or if you opt to install your packages globally.   In general, we should avoid 'shelling out' whenever possible (tests and other internal tools being the exception).